src/scripts/breeding/Breeding.ts

```diff
@@ -266,6 +266,15 @@ class Breeding implements Feature {
             if (this.queueList().length) {
                 const nextEgg = this.createEgg(this.queueList.shift());
                 this.gainEgg(nextEgg);
+                if ((this.queueSlots() - this.queueList().length) === this.queueSlots()) {
```

Comment:
Not sure as I haven't fully tested, but aren't we able to simplify this by just checking if the queue is now empty?
```suggestion
                if (!this.queueList().length) {
```

Reply:
If you're saying so I am sure we can. I'll commit it now.